string robot_name
uint8 id
geometry_msgs/PoseWithCovariance pose
float32[] descriptor
float32[] navigableDirection
BinaryArrayMsg[] frontierPoints
float32[] frontierDistance